### PR TITLE
Cydnee Owens

### PR DESCRIPTION
**Pull request by cowens87**
Iteration 1 Complete (All tests passed)
Iteration 2 Complete (All tests passed)
Iteration 3 Almost complete (last two assertions "In progress
Iteration 4 Semi Complete (testing setup per IP, but no code)

### CODE REVIEW

File: lib/house.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ attr_reader :price, :address, :rooms
+
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+  end
+
+  def price
+    price = "#{@price.delete "$"}".to_i
```
Comment (mikedao) [inline lib/house.rb:11]: Good attention to detail.

File: lib/house.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    @rooms << room
+  end
+
+  def above_market_average?
+    self.price > 500000
+  end
+
+  def rooms_from_category(category)
+    rooms.find_all do |room|
+     room.category == category
```
Comment (mikedao) [inline lib/house.rb:24]: Great

File: lib/house.rb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  def area
+    rooms.find_all do |room|
+     room.area += 1
+     #sum = the sum of area of all rooms
+   end
+  end
+
+  def detals
+  house.details[price, address]
+  end
```
Comment (mikedao) [inline lib/house.rb:37]: Watch your spelling and indentation here.

**Comment by cowens87** [comment]
> Progress over last week, what you have done is really good. I would suggest that you reach out to Krista and book some time, I will have Krista contact you.
> 
> Score: 2

Hey Mike, 
Thank you so much for the feedback and I would love to connect with Krista. I do have a question on the scoring. I know I didn't make it all the way through iteration 3, but do I not get any points for the assertions I did do? I'm just trying to understand how these are evaluated.

**Comment by mikedao** [comment]
It's based on how many you clear. We score Independent Challenges differently from paired evals or projects. A 3 would have been if you had gotten all of I3. It would have been a 2.5 if you had gotten #area done.

**Pull request closed without merging**